acs-apiclient/AcsApiClientConfig.cs

```diff
@@ -128,7 +139,7 @@ public class AcsApiClientConfig
         ) {
             this.ConsumerKey = consumerKey;
             this.ConsumerSecret = consumerSecret;
-            this.ServerRoot = serverRoot.EndsWith("/") ? serverRoot : serverRoot + "/";
+            this.ServerRoot = serverRoot.EndsWith("/", StringComparison.CurrentCulture) ? serverRoot : serverRoot + "/";
```

Comment:
This is more a question for me since you purposefully added this: Whats the reason for using `StringComparison.CurrentCulture`?

Reply:
As discussed with Ahmad: This was added according to the warning displayed in Xamarin.

Reply:
:white_check_mark: